### PR TITLE
fix(test): update used filter transient test

### PR DESCRIPTION
**Pull request by Levdbas**
## Issue
Transient test still uses deprecated old filter

## Solution
Let's fix that

## Impact
Better tests

## Usage Changes
no

## Considerations
no

## Testing
no

### CODE REVIEW

File: tests/test-timber-cache.php [hunk 1]
```diff
@@ -179,9 +179,9 @@ public function testTransientForceFilter()
 
         Timber\Helper::_lock_transient($transient, 30);
 
-        add_filter('timber_force_transient_' . $transient, '__return_true');
+        add_filter('timber/transient/force_transient_' . $transient, '__return_true');
```
Comment (gchtr) [inline tests/test-timber-cache.php]:
```suggestion
        $this->add_filter_temporarily('timber/transient/force_transient_' . $transient, '__return_true');
```

File: tests/test-timber-cache.php [hunk 2]
```diff
@@ -179,9 +179,9 @@ public function testTransientForceFilter()
...
         $get_transient = Timber\Helper::transient($transient, '__return_true');
-        remove_filter('timber_force_transient_' . $transient, '__return_true');
+        remove_filter('timber/transient/force_transient_' . $transient, '__return_true');
```
Comment (gchtr) [inline tests/test-timber-cache.php]:
```suggestion
```

**Review by gchtr — CHANGES_REQUESTED** [review]
Let’s use `add_filter_temporarily()` when we already touch it :).

**Review by gchtr — DISMISSED** [review]

**Comment by Levdbas** [comment]
Alright, we are good to go here @gchtr . This should fix the failed tests in https://github.com/timber/timber/pull/2898 as well

**Review by gchtr — APPROVED** [review]
Loooking goooood! 😎

**Pull request merged**